app/menus/DocumentMenu.tsx

```diff
@@ -48,75 +50,120 @@ import {
   moveTemplate,
 } from "~/actions/definitions/documents";
 import useActionContext from "~/hooks/useActionContext";
+import useBoolean from "~/hooks/useBoolean";
 import useCurrentUser from "~/hooks/useCurrentUser";
 import useMobile from "~/hooks/useMobile";
 import usePolicy from "~/hooks/usePolicy";
 import useRequest from "~/hooks/useRequest";
 import useStores from "~/hooks/useStores";
 import { MenuItem } from "~/types";
 import { documentEditPath } from "~/utils/routeHelpers";
+import { MenuContext, useMenuContext } from "./MenuContext";
 
 type Props = {
+  /** Document for which the menu is to be shown */
   document: Document;
-  className?: string;
```

Comment:
I don't think we need this yet? Not used at any of the call sites of `DocumentMenu`.